torch/testing/_internal/common_nn.py

```diff
@@ -102,6 +102,7 @@ def get_weight(m):
     dict(
         module_name='Linear',
         constructor_args=(10, 8),
+        constructor_kwargs=dict(dtype=torch.double),
```

Comment:
cc @jbschlosser -- you probably want these to be constructed in float not double?